hail/src/test/scala/is/hail/expr/ir/IRSuite.scala

```diff
@@ -1834,4 +1834,135 @@ class IRSuite extends SparkSuite {
 
     assertEvalsTo(ir, 61L)
   }
+
+  @Test def testfoo() {
```

Comment:
Not a super huge fan of this as a test---do we need it? (At the very least can we rename it?)

Reply:
oops meant to delete it